munigeo_migrations/0003_migrate_translations_to_parler.py

```
@@ -0,0 +1,63 @@
+        (AD, ADTranslation),
+        (Municipality, MunicipalityTranslation),
+        (Street, StreetTranslation),
+    ]
+
+
+def forwards_func(apps, schema_editor):
+    for lang_code, _ in settings.LANGUAGES:
+        name_field_key = "name_" + lang_code
+
+        for Model, ModelTranslation in _get_munigeo_models(apps):  # noqa: N806
+            for object in Model.objects.all():
```

Ruff (0.15.15)

[error] Variable `object` is shadowing a Python builtin

(A001)